packages/design-tokens/tokens/color/core.json

```diff
@@ -65,6 +72,7 @@
     "deep-purple": { "value": "#190f33" },
     "action-purple": { "value": "#b921f1" },
     "light-purple": { "value": "#f7d6ff" },
-    "electric-purple": { "value": "#6933ff" }
+    "electric-purple": { "value": "#6933ff" },
+    "electric-teal": { "value": "#58ffd2" }
```

Comment:
aaaaand https://www.youtube.com/watch?v=MmZexg8sxyk is stuck in my head

Reply:
we can also thank MGMT for providing the theme song for 2 of my internships https://www.youtube.com/watch?v=OTHHeIAYfuU

["If I had a nickel for every-time an internship mentor was named Michael, I'd have 2 nickels, which really isn't a lot but it's weird that it happened twice"](https://www.youtube.com/watch?v=Ejcpbytf-Dg)